### ISSUE TITLE
Wrong dead soldier sprites - ECWOlf 1.2.2 with Tech Demo

### ISSUE THREAD

**Issue by Blzut3** [ECWolfEngine/ECWolf#91]
**[Original report](https://bitbucket.org/ecwolf/ecwolf/issues/91) by Anonymous.**

----------------------------------------

When using ECWolf with the techdemo12.pk3 file on the original Wolf3D wl6 files, the standard soldiers wearing grey revert to their original brown uniform sprite when they die. They also show the original brown sprite when walking in certain directions. Don't know if this is an error with the Tech Demo or the new release, though.

Note: Bug tracker won't let me select version 1.2.2.

**Comment by Blzut3** [comment]
**Original comment by Anonymous.**

----------------------------------------

I'd like to update this, as it turns out I wasn't using the Tech Demo correctly. What I'd done was load a Wolf3D save file instead of starting a new game, meaning that some of the sprites (dead soldiers and doors are those I noticed) were switching.

I've now played through all 9 maps of the Tech Demo and it played perfectly. The only error I noticed was a misspelling in the end of game text - hehind instead of behind.
